### PR TITLE
Add bob.bio.base recipe

### PR DESCRIPTION
**Pull request by 183amir**

### CODE REVIEW

**Comment by 183amir** [comment]
This depends on https://github.com/conda-forge/staged-recipes/pull/409

**Comment by 183amir** [comment]
LGTM?

File: recipes/bob.bio.base/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,113 @@
+{% set version = "2.0.7" %}
+
+package:
+  name: bob.bio.base
+  version: {{ version }}
+
+source:
+  fn: bob.bio.base-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/b/bob.bio.base/bob.bio.base-{{ version }}.zip
```
Comment (msarahan) [inline recipes/bob.bio.base/meta.yaml]: Should replace pypi.python.org with pypi.io

**Comment by msarahan** [comment]
+1 for merge after pypi URL change

**Comment by jakirkham** [comment]
Thanks @183amir.

**Pull request merged**